libs/packages/components/src/lib/search-result-list/search-result-list.component.ts

```diff
@@ -1,15 +1,23 @@
-import { Component, Input, ContentChild, TemplateRef } from '@angular/core';
+import {
+  Component,
+  Input,
+  ContentChild,
+  TemplateRef,
+  OnInit,
+  OnChanges
+} from '@angular/core';
 @Component({
   selector: 'sds-search-result-list',
   templateUrl: './search-result-list.component.html',
   styleUrls: ['./search-result-list.component.scss']
 })
-export class SdsSearchResultListComponent {
-
+export class SdsSearchResultListComponent
+  implements OnInit, OnChanges {
+  constructor() {}
   /**
    * List of items
    */
-  @Input() model: any[];
```

Comment:
model should have a type that matches with what display template is using

Reply:
The component is receiving 2 different types. An array assignment or object assignment would cause everything to break.

Reply:
They can have optional params

Reply:
If you have a reference that works, I can make the change. At the moment if I assign types, it throws terminal errors and breaks the build.